vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/MapEventsPage.java

```diff
@@ -34,6 +34,13 @@ public MapEventsPage() {
             eventData.setText(eventDataText);
         });
 
+        map.addMapClickEventListener(event -> {
+            eventData.setText(event.getCoordinate().getX() + ";"
+                    + event.getCoordinate().getY());
+            viewState.setText(event.getMouseDetails().getAbsoluteX() + ";"
+                    + event.getMouseDetails().getAbsoluteY());
```

Comment:
The view state div has a different purpose in another test, we should not store the pixel values in it. Would suggest to just append the pixel values to the event data div.

Reply:
Done.